openedx/core/djangolib/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+from django.core.cache import caches
+from django.test import TestCase, override_settings
+
+
+class CacheSetupMixin(TestCase):
```

Comment:
Hm. I'd like it to be as easy as possible to add it into a new test suite, though. Also, I think clearing the cache between every test is still more correct, even though the base database is shared between the tests.

Objections to me leaving it as-is until/unless we see refilling the cache being a performance bottleneck?